src/Server/TextDocument.php

```diff
@@ -98,7 +98,7 @@ public function didOpen(TextDocumentItem $textDocument)
      */
     public function didChange(VersionedTextDocumentIdentifier $textDocument, array $contentChanges)
     {
-        $this->project->getDocument($textDocument->uri)->updateContent($contentChanges[0]->text);
+        $this->project->openDocument($textDocument->uri, $contentChanges[0]->text);
```

Comment:
You are messing a bit here with the semantics. `didChange` is not supposed to _open_ a document, it is supposed to _update_ an opened document.